packages/patternfly-4/react-core/src/components/Switch/Switch.d.ts

```diff
@@ -0,0 +1,17 @@
+import { HTMLProps, FormEvent, ReactNode } from 'react';
+import { Omit } from '../../typeUtils';
+
+export interface SwitchProps extends Omit<HTMLProps<HTMLInputElement>, 'type' | 'onChange' | 'disabled' | 'label'> {
+  isDisabled?: boolean;
+  isChecked?: boolean;
```

Comment:
Feel like this should be value to be consistent with PF3, and the other components in PF4.

Reply:
Also if we are going to support tri-state we will need to use something other than a boolean.  Maybe constants?

Reply:
http://patternfly-react.surge.sh/patternfly-3/index.html?knob-Size=normal&knob-On%20Color=primary&knob-Off%20Color=default&knob-animate=true&selectedKind=patternfly-react%2FWidgets%2FSwitch&selectedStory=Switch&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Fstories%2Fstories-panel

Reply:
I just looked at PF3.  It looks like it uses react-bootstrap-switch for the implementation.  react-bootstrap-switch allows for the consumer to turn triState on.  That allows for a state of null or the indeterminate state.  I think the question there are 2 questions here
1) Do rename the prop to "value" to be consistent with PF3
2) Will we be allowing for the indeterminate state to align with PF3.  If we do support that we need to add another prop to turn it o/ off like react-bootsrap-switch does.

Reply:
patternfly-react should be the react implementation of patternfly-core.
So if triState is not in patternfly-core, it should not be here

for me you should open an issue on pf-core first